### ISSUE TITLE
When checking the status of a dataset, if there is no import control record...

### ISSUE THREAD

**Issue by Foxcapades** [VEuPathDB/vdi-service#44]
default the value to awaiting import.

This is due to the fact that we don't do anything with a dataset on first posting so there will be no import control record for it before the import starts.

**Comment by Foxcapades** [comment]
Blocked by #74

**Comment by Foxcapades** [comment]
There will also be no dataset record, so we will just need to accommodate the 404 when we first post the dataset.

Creating a new ticket to update the API docs to document this.
